src/components/icons/unlink-icon.vue

```diff
@@ -2,11 +2,21 @@
   - Copyright (c) TIKI Inc.
   - MIT license. See LICENSE file in root directory.
   -->
+<script setup lang="ts">
+defineEmits(["click"]);
+defineProps({
+  syncAccount: {
+    type: Boolean,
+    required: false,
+    default: false,
+  },
+});
+</script>
 
 <template>
   <svg
     viewBox="0 0 100 100"
-    fill="currentColor"
+    :fill="!syncAccount ? '#C73000' : 'currentColor'"
```

Comment:
icon components should not contain any properties, emits, or otherwise— in fact, they could be moved to pure svg files leveraging a vite plugin. 

logic should always be handled at the parent component layer.

Reply:
Morning, Mike.

I've created another icon-component, the error-icon. then used v-if and v-else to load the correct icon

also added the verified property in ReceiptAccount class